### PR TITLE
Prompt the user for touch on Linux systems

### PR DESCRIPTION
**Pull request by antoinealb**
Related to #8 (Prompt for touch). Note that it only works on Linux sytems using libnotify.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -320,3 +323,12 @@ func (a *Agent) Lock(passphrase []byte) error {
 func (a *Agent) Unlock(passphrase []byte) error {
 	return ErrOperationUnsupported
 }
+
+func notifyTouchRequired() {
+
+	if _, err := exec.LookPath("notify-send"); err == nil {
+		cmd := exec.Command("notify-send", "-a", "yubikey-agent", "Touch required", "Please touch your Yubikey for SSH")
```
Comment (exussum12) [inline main.go:330]: worth adding `-i info` to this ? 
image vs 
image

Comment (antoinealb) [reply main.go:330]: I don't think there is a standard list of icons guaranteed to be available on all distributions. If "icon" is guaranteed to be available, then why not yes.

Comment (exussum12) [reply main.go:330]: The standard flag is `dialog-information`  https://specifications.freedesktop.org/icon-naming-spec/latest/ar01s04.html  . thats probably a better choice.

Comment (exussum12) [reply main.go:330]: or `dialog-password` ?

Comment (antoinealb) [reply main.go:330]: Dialog-password is good for me. Added it to the code.

**Comment by exussum12** [comment]
Been using this some more, using the instructions linked in the README, the systemd file doesnt have the correct DISPLAY and XAUTHORITY required for notify-send to work.
That file has actually been deleted too, new instructions would be needed to run with a user systemd unit, rather than the system location (which the docs suggest)

See https://wiki.archlinux.org/index.php/Systemd/User#DISPLAY_and_XAUTHORITY for more info

**Comment by antoinealb** [comment]
I have been using this patch with this systemd user unit: https://aur.archlinux.org/cgit/aur.git/tree/yubikey-agent.service?h=yubikey-agent and it worked fine.

**Comment by exussum12** [comment]
That is correct (https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=yubikey-agent) uses a --user, the docs of this repo link to a system unit file. I think the install from AUR needs adding here for people not on arch.

**Pull request closed without merging**